snap/validate_test.go

```diff
@@ -69,6 +71,14 @@ func (s *ValidateSuite) TestValidateName(c *C) {
 	invalidNames := []string{
 		// name cannot be empty
 		"",
+		// names cannot be full
```

Comment:
I am probably missing something but what does "full" mean here?

Reply:
just a joke when compared to the "empty" above